Engines/Wine/QuickScript/Steam Script/script.js

```diff
@@ -114,23 +132,36 @@ module.default = class SteamScript extends QuickScript {
         new Corefonts(wine).go();
 
         // Steam must be started once such that config.vdf is created (see fixCertificateIssue())
-        setupWizard.wait(tr("Please follow the steps of the Steam setup. Then, wait until Steam is updated, log in and finally close Steam completely so the installation of \"{0}\" can continue.", this._name));
+        setupWizard.wait(
+            tr(
+                'Please follow the steps of the Steam setup. Then, wait until Steam is updated, log in and finally close Steam completely so the installation of "{0}" can continue.',
+                this._name
+            )
+        );
         wine.run(tempFile, [], null, false, true);
 
         // Set windows environment for executable that needs it
-        wine.setOsForApplication().set("steam.exe", "winxp").do();
-        wine.setOsForApplication().set("steamwebhelper.exe", "winxp").do();
+        wine.setOsForApplication()
+            .set("steam.exe", "winxp")
```

Comment:
@ImperatorS79 is `winxp` still the right setting here? If you open steam with this it will show a warning label that winxp is not supported anymore